src/core/public/application/ui/app_container.tsx

```diff
@@ -77,12 +80,13 @@ export const AppContainer: FunctionComponent<Props> = ({
           element: elementRef.current!,
           onAppLeave: handler => setAppLeaveHandler(appId, handler),
         })) || null;
+      setIsMounting(false);
```

Comment:
should we try/catch `mounter.mount` to `setIsMounting(false)` even in case of error? Wondering if it's really useful though.